src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -227,21 +235,24 @@ describe('CdkVirtualScrollViewport', () => {
     it('should set content offset to top of content', fakeAsync(() => {
       finishInit(fixture);
       viewport.setRenderedContentOffset(10, 'to-start');
+      fixture.changeDetectorRef.markForCheck();
       fixture.detectChanges();
       flush();
 
       expect(viewport.getOffsetToRenderedContentStart()).toBe(10);
     }));
 
-    it('should set content offset to bottom of content', fakeAsync(() => {
+    // TODO: verify
+    fit('should set content offset to bottom of content', fakeAsync(async () => {
```

Comment:
Verify: replacing `detectChanges()` with `markForCheck(); detectChanges()` wasn't enough, had to switch to `autoDetectChanges()` / `whenStable()`

Reply:
You can remove `autoDetect` here